internal/locate/region_cache.go

```diff
@@ -3164,6 +3339,10 @@ func (s *Store) recordReplicaFlowsStats(destType replicaFlowsType) {
 	atomic.AddUint64(&s.replicaFlowsStats[destType], 1)
 }
 
+func (s *Store) recordHealthFeedback(feedback *tikvpb.HealthFeedback) {
+	s.healthStatus.updateTiKVServerSideSlowScore(int64(feedback.GetSlowScore()), time.Now())
```

Comment:
feedback.FeedbackSeqNo is never used now?

Reply:
Yes, it's not checked for now. Maybe I'd better add a comment to note it.